doc/installation.rst

```diff
@@ -34,25 +38,36 @@ Just visit the `Spyder page on Binder`_ to get started using Spyder now.
 
 
 
-.. _standalone_installers_ref:
+.. _install-standalone:
 
 =====================
 Standalone installers
 =====================
 
-Our standalone installers for Windows and macOS are available from Spyder 4.2 onwards.
-We recommend using this installation method on those platforms, but we offer several other options for Linux, advanced users and specific needs, so keep reading if that's the case for you.
+The standalone installers are our recommended method for most users on Windows and macOS, with experimental Linux support under active development.
+They work much like for any other IDE, where Spyder can be installed and updated independently of the Python environments you use to run your code.
```

Comment:
```suggestion
They work similar to any other IDE, where Spyder can be installed and updated independently of the Python environments you use to run your code.
```

Reply:
Technically, the revised version implies that the installers work similar to other IDEs themselves rather than those IDEs' _installers_. If we're going to go this route, may as well make it even simpler:

```suggestion
They work like any other IDE, where Spyder can be installed and updated independently of the Python environments you use to run your code.
```